Modifying example to search command subfolders (#1486)
discordjs/guide: guide/additional-features/reloading-commands.md

The reload command ideally should not be used by every user. You should deploy it as a guild command in a private guild.
:::

> We will use the glob package to navigate our slash command file structure. This can be installed via npm by running `npm i glob` in the terminal.

Reviewer: ```suggestion
You can use the glob package to navigate your slash command file structure. To install the package run `npm install glob` in the terminal.
```
wording and consistency https://github.com/discordjs/guide/blob/main/CONTRIBUTING.md#youyour-instead-of-weour
